source/_docs/configuration/templating.markdown

```diff
@@ -76,6 +76,8 @@ Not supported in [limited templates](#limited-templates).
 - `is_state('device_tracker.paulus', 'home')` will test if the given entity is the specified state.
 - `state_attr('device_tracker.paulus', 'battery')` will return the value of the attribute or None if it doesn't exist.
 - `is_state_attr('device_tracker.paulus', 'battery', 40)` will test if the given entity attribute is the specified state (in this case, a numeric value). Note that the attribute can be `None` and you want to check if it is `None`, you need to use `state_attr('sensor.my_sensor', 'attr') == None`. 
+- `is_nominal('sensor.my_sensor')` will test if the given entity exist and is not unknown or unavailable. A list of entities can be provided instead of a string. Can be used as a filter. `nominal` is the test equivalent.
```

Comment:
After the short discussion in the core PR: Would it make sense to explicitly mention that for a list of entities an OR logic is applied!? Cheers